java/src/main/java/com/saucelabs/simplesauce/SauceSession.java

```diff
@@ -118,18 +115,19 @@ private MutableCapabilities setRemoteDriverCapabilities(MutableCapabilities sauc
         return currentSessionCapabilities;
     }
 
-    private void tryToCreateRemoteWebDriver(String sauceLabsUrl) {
+    RemoteWebDriver createRemoteWebDriver() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4730723)